src/components/menu/menu/menu.component.html

```diff
@@ -2,6 +2,8 @@
     <div
         *ngIf="isMenuOpen"
         class="ux-menu"
+        role="menu"
+        [id]="id"
```

Comment:
I mentioned that this should be derived from the provided `id` input rather than applied directly. The reason for that is to avoid duplicate ids in the DOM:
![image](https://user-images.githubusercontent.com/26114101/125935510-10ed046f-ecea-4cb7-8df3-95ca0cc15d0c.png)